### DISCUSSION TITLE
softmax in UNET

### DISCUSSION THREAD

**Discussion by amida47** [Project-MONAI/tutorials#1715] [Q&A]
is the UNET in monai.networks.nets.UNet does perform a softmax at end, or should I manually apply it to the predictions??

**Comment by KumoLiu** [comment]
Hi @amida47, please manually add it using `Activations` transform.
Please refer to:
https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/brats_segmentation_3d.ipynb

Thanks.

**Reply by amida47** [reply]
I would like to suggest that a hint to the fact that the model doesn't do the softmax on it's own to the API reference, I was completely oblivious to to this and I went troubleshooting something else.
